src/index.html

```diff
@@ -6,6 +6,9 @@
 	<base href="/">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="icon" type="image/x-icon" href="favicon.ico">
+	<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
These libraries aren't needed. Refrain from using jquery as it's not recommended in combination with angular 2. Masonry has already been added to the project as directives. Check for reference in the article-grid-component.ts (starting line 102) and html files [https://github.com/osqledaren/web1617/tree/master/src/app/article-grid](url).

Documentation is found here [https://www.npmjs.com/package/angular2-masonry](url)